build/yaml/steps/codeanalysis/prefast.yaml

```diff
@@ -9,3 +9,5 @@ steps:
   displayName: 'Install Prefast Plugin'
 
 - template: ../windows/binaries.yaml

```

Comment:
How is this able to build without restoring the NuGet packages?

Reply:
binaries.yaml restores the nuget packages.